core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorTableMetadata.java

```diff
@@ -72,13 +83,26 @@ public Optional<String> getComment()
         return comment;
     }
 
+    /**
+     * List of constraints data in a table is expected to satisfy.
+     * Engine ensures rows written to a table meet these constraints.
+     * A check constraint is satisfied when it evaluates to True or Unknown.
+     * @return List of string representation of a Trino SQL scalar expression that can refer to table columns by name and produces a result coercible to boolean
+     */
+    @Experimental(eta = "2023-03-31")
+    public List<String> getCheckConstraints()
```

Comment:
Where does this method get used? I see no usage of this method at the moment.

Reply:
Removed this method and moved the comment to `ConnectorTableSchema`.